src/test/java/edu/nextstep/camp/racingcar/step3/ResultViewTest.java

```diff
@@ -0,0 +1,42 @@
+package edu.nextstep.camp.racingcar.step3;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ResultViewTest {
+    private final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+
+    @BeforeEach
+    void setUp() {
+        System.setOut(new PrintStream(outputStream));
+    }
+
+    @Test
+    void 자동차의_현재_위치가_잘_출력되는지_테스트() {
+        // Given
+        List<Car> cars = Arrays.asList(new Car(2), new Car(1), new Car(3));
```

Comment:
```suggestion
        List<Car> cars = List.of(new Car(2), new Car(1), new Car(3));
```

`List.of`와 어떤 차이가 있을까요?

Reply:
Arrays.asList는 완전 불변은 아니라 값 수정은 가능하고, List.of는 완전 불변하도록 최적화가 되어 있다고 하네요. immutable하게 이용하기 위해서는 List.of를 이용하는 것이 나을 것 같아 수정해 두겠습니다!